controllers/index.js

```diff
@@ -1,9 +1,15 @@
 const router = require('express').Router();
 
 const apiRoutes = require('./api');
-const homeRoutes = require('./homeRoutes');
+const homeRoutes = require('./homeRoutes.js');
+const profileRoutes = require('./profileRoutes.js');
 
 router.use('/', homeRoutes);
 router.use('/api', apiRoutes);
+router.use('/profile', profileRoutes);
+
+router.use((req, res) => {
```

Comment:
What is this?  They didn't have this in the crowd funding mini-project

Reply:
This is something I pulled from an example.  If you think it's not needed, we can omit it. It's meant to send an error message.